vita3k/renderer/src/vulkan/scene.cpp

```diff
@@ -99,6 +99,18 @@ void update_sync_signal(SceGxmSyncObject *sync) {
     }
 }
 
+#ifdef __APPLE__
+void restride_stream(GXMStreamInfo &stream, uint32_t old_stride) {
+    const uint32_t new_stride = (old_stride + 3) & -4;
+    stream.size = (stream.size / old_stride) * new_stride;
+    uint8_t *new_data = new uint8_t[stream.size]();
```

Comment:
```suggestion
    uint8_t *new_data = new uint8_t[stream.size];
```

No need to initialize new_data